x-pack/plugins/encrypted_saved_objects/server/crypto/encrypted_saved_objects_service.test.ts

```diff
@@ -54,12 +51,6 @@ describe('#registerType', () => {
       service.registerType({ type: 'known-type-1', attributesToEncrypt: new Set(['attr']) })
     ).toThrowError('The "known-type-1" saved object type is already registered.');
   });
-
-  it('throws if `type` references to the unknown type', () => {
```

Comment:
__note:__ this check didn't bring lots of value and it doesn't fit very well to NP since plugins will register their types during `setup` that would make order of these operations important. So since SO client will return error for not known type I decided to ditch this check. What do you think?